index.html

```diff
@@ -1133,6 +1133,55 @@ <h3>Revocation Bitstring Length</h3>
       </p>
     </section>
 
+    <section class="informative">
+      <h3>Unnecessary Correlation</h3>
+
+      <p>
+There are a number of global identifiers used in a status list entry, defined in
+Section [[[#bitstringstatuslistentry]]], that can be used across [=verifiers=]
+to correlate [=subjects=]. Some of these properties include `id`,
+`statusListIndex`, and `statusListCredential`.
+      </p>
+
+      <p>
+In some cases, when [=presenting=] a [=verifiable credential=] that contains
+other global identifiers (such as a driver's license identification number),
+using other global identifiers for the status list information does not cause
+further correlation harms since the only thing required for correlation is a
+single globally unique identifier.
+      </p>
+
+      <p>
+When these global identifiers are used in [=presentations=] that utilize
+[=selective disclosure=] or [=unlinkable disclosure=], they can violate privacy
```

Comment:
It seems that respec cannot locate the [=unlinkable disclosure=] term. This should be handled

Reply:
I added a terminology placeholder in the VCDM spec yesterday so that we can refer to it (since that spec has had the concept of unlinkable disclosure since v1.0). It takes 6 hours for the terminology database to update after the TR spec is published.

It looks like the terminology database updated and the link works now: https://www.w3.org/TR/vc-data-model-2.0/#dfn-unlinkable-disclosure